velox/dwio/dwrf/writer/Writer.cpp

```diff
@@ -32,6 +32,11 @@
 using facebook::velox::common::testutil::TestValue;
 
 namespace facebook::velox::dwrf {
+
+void WriterOptions::processConfigs(
```

Comment:
Consider to move the connector and session properties init into this function? Thanks!